### PR TITLE
CredentialProvider UI Polish

### PR DESCRIPTION
**Pull request by rilury**
- Redesign UI for light and dark mode
- The strings format was changed
- The new strings were added as localized strings with the default English value
- Custom spinner from Welcome screen was removed
- Hide navigation bar during search also for iPad
- Default activity indicator was added to Welcome screen

| Light mode | Dark mode |
| ------------- | ------------- |
| Simulator Screen Shot - iPhone 12 Pro Max - 2021-07-09 at 12 17 07| Simulator Screen Shot - iPhone 12 Pro Max - 2021-07-12 at 15 46 41|
| Simulator Screen Shot - iPhone 12 Pro Max - 2021-07-09 at 12 25 27|  Simulator Screen Shot - iPhone 12 Pro Max - 2021-07-12 at 15 47 00|
| Simulator Screen Shot - iPhone 12 Pro Max - 2021-07-09 at 12 26 48|  Simulator Screen Shot - iPhone 12 Pro Max - 2021-07-12 at 15 45 50|
|Simulator Screen Shot - iPhone 12 Pro Max - 2021-07-12 at 16 57 55|Simulator Screen Shot - iPhone 12 Pro Max - 2021-07-12 at 16 57 00|

### CODE REVIEW

File: Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -1386,16 +1386,13 @@ extension String {
...
-    public static let LoginsListNoMatchingResultTitle = MZLocalizedString("LoginsList.NoMatchingResult.Title", tableName: "CredentialProvider", comment: "Label displayed when a user searches and no matches can be found against the search query")
-    public static let LoginsListNoMatchingResultSubtitle = MZLocalizedString("LoginsList.NoMatchingResult.Subtitle", tableName: "CredentialProvider", comment: "Label that appears after the search if there are no logins available to this account")
-    public static let LoginsListNoLoginsFoundTitle = MZLocalizedString("LoginsList.NoLoginsFound.Title", tableName: "CredentialProvider", comment: "Label informing the user the account has no logins available")
-    public static let LoginsListNoLoginsFoundSubtitle = MZLocalizedString("LoginsList.NoLoginsFound.Subtitle", tableName: "CredentialProvider", comment: "Label shown when there are no logins to list")
+    public static let WelcomeViewTitle = MZLocalizedString("WelcomeView.Title", value: "Take your passwords everywhere", comment: "Label displaying welcome view title")
```
Comment (st3fan) [inline Client/Frontend/Strings.swift:1389]: @rilury Can you please rename this to `LoginsWelcomeViewTitle` then it is a bit less generic and clear that it is part of the same set of strings for logins.

**Review by st3fan — CHANGES_REQUESTED** [review]

File: Client/Frontend/Strings.swift (hunk 1 above)
Comment (rilury) [reply Client/Frontend/Strings.swift:1389]: Sure, I will rename it to LoginsWelcomeViewTitle 👍

**Review by st3fan — APPROVED** [review]
This looks good to me - let's land it 🙏

**Comment by SimonBasca** [comment]
Verified fix on credential provider branch faaf9b8bd928199680cecd7cfe67a95816190077.

**Pull request merged**